src/main/java/org/fcrepo/spec/testsuite/crud/Container.java

```diff
@@ -729,4 +732,571 @@ public void ldpDirectContainerMayAllowLdpIsMemberRelationToBeUpdatedByPatch() {
+            throw new SkipException(
+                "This implementation does not support PATCH updates on " + LDP_IS_MEMBER_OF_RELATION_PREDICATE);
+        }
+
+        final Response getUpdatedResource = doGet(containerResource);
+        if (!testForPresenceOfTrip(getUpdatedResource, containerResource,
+                                   LDP_IS_MEMBER_OF_RELATION_PREDICATE, isMemberOfPredicate2, true)) {
+            throw new SkipException(
+                "This implementation does not support PATCH updates on " + LDP_IS_MEMBER_OF_RELATION_PREDICATE);
+        }
+
+        confirmPresenceOrAbsenceOfTripleInResponse(getUpdatedResource, containerResource,
+                                                   LDP_IS_MEMBER_OF_RELATION_PREDICATE, isMemberOfPredicate1, false);
+
+    }
+
+    /**
+     * 3.1.3-N
+     */
+    @Test(groups = {"MUST"})
+    public void ldpIndirectContainerMustAllowInsertedContentRelationPredicateToBeSetOnCreate() {
+        setupTest("3.1.3-N",
+                  "Implementations must allow the ldp:insertedContentRelation property to be set via the content RDF " +
+                  "on container creation",
+                  "https://fcrepo.github.io/fcrepo-specification/#ldpic",
+                  ps);
+
+        //throw skip exception if indirect containers not supported
+        skipIfIndirectContainersNotSupported();
+
+        final String memberSubject = "http://example.org/ldp/MemberSubject";
+        final String body = "<> <" + LDP_INSERTED_CONTENT_RELATION_PREDICATE + "> <" + memberSubject + "> ;";
+        final Response container = createIndirectContainer(uri, body);
+        final String containerResource = getLocation(container);
+        confirmPresenceOrAbsenceOfTripleInResponse(doGet(containerResource), containerResource,
+                                                   LDP_INSERTED_CONTENT_RELATION_PREDICATE, memberSubject, true);
+    }
+
+    /**
+     * 3.1.3-O
+     */
+    @Test(groups = {"MUST"})
+    public void ldpIndirectContainerMustAllowInsertedContentRelationToBeSetByDefault() {
+        setupTest("3.1.3-O",
+                  "Implementations must allow the ldp:insertedContentRelation property to be set by default to an " +
+                  "implementation defined value.",
+                  "https://fcrepo.github.io/fcrepo-specification/#ldpic",
+                  ps);
+
+        //throw skip exception if indirect containers not supported
+        skipIfIndirectContainersNotSupported();
+
+        final String body = "";
+        final Response container = createIndirectContainer(uri, body);
+        final String containerResource = getLocation(container);
+        final String responseBody = doGet(getLocation(container)).getBody().asString();
+        try (CloseableDataset dataset = parseTriples(IOUtils.toInputStream(responseBody))) {
+            final DatasetGraph graph = dataset.asDatasetGraph();
+            if (!graph
```

Comment:
Why not use `confirmPresenceOrAbsenceOfTripleInResponse` here?

Reply:
Because I don't know in this case what the object of the triple will be.  It is implementation-specific.